src/Template/Variable.php

```diff
@@ -24,6 +24,11 @@ public function getName(): string
         return $this->name;
     }
 
+    public function getType(): Type
```

Comment:
do we need this? it is not used

Reply:
shit it was commited by mistake. I need it in that variable type merging I was working on and I commited it in this PR by mistake. Should I remove it or will we keep it here?